### PR TITLE
Fix configs using wrong otp applicaitons

### PR DESCRIPTION
**Pull request by rhnonose**
Some configurations were using the main `re` application as otp app and some apps were being launched under `re` as well.
Refactor to use appropriate `re_web` and `re_integrations` applications

### CODE REVIEW

File: apps/re_integrations/lib/pipedrive/pipedrive.ex [hunk 1]
```diff
@@ -5,7 +5,7 @@ defmodule ReIntegrations.Pipedrive do
   require Logger
 
   @activity_params ["deal_id", "marked_as_done_time"]
-  @server Application.get_env(:re, :pipedrive, __MODULE__.Server)
+  # @server Application.get_env(:re_integrations, :pipedrive, __MODULE__.Server)
```
Comment (joaodubas) [inline apps/re_integrations/lib/pipedrive/pipedrive.ex]: Why comment this line?

Comment (rhnonose) [reply apps/re_integrations/lib/pipedrive/pipedrive.ex]: This was sort of a "mock" to not call it but it wasn't being used (the test module in `test.exs` wasn't event declared), so I tested removing it but forgot to remove the commented code.

File: config/dev.secret-example.exs [hunk 2]
```diff
@@ -22,20 +22,22 @@ config :re, ReWeb.Guardian,
     "y" => "xxxxxx"
   }
 
-# The emails below are used when a visitor
-# fills out a form expressing interest in a listing.
-# Both `to:` and `from:` can be individual emails
```
Comment (pmargreff) [inline config/dev.secret-example.exs]: This comment (mainly the part about how to fill the emails vars) looks informative could be a good idea keep it (if you don't mind, of course).

File: config/test.exs [hunk 3]
```diff
@@ -37,22 +33,21 @@ config :re, Re.Repo,
 config :account_kit,
   app_id: "123"
 
-config :re, ReIntegrations.Notifications.Emails.Mailer, adapter: Swoosh.Adapters.Test
-
-config :re, :visualizations, Re.TestVisualizations
-config :re, :emails, ReIntegrations.Notifications.Emails.UserTest
-config :re, :elasticsearch, ReWeb.TestSearch
-config :re_integrations, :http, ReIntegrations.TestHTTP
-config :re, :pipedrive, ReWeb.TestPipedriveServer
```
Comment (pmargreff) [inline config/test.exs]: Didn't find this module to be excluded, but I guess it was excluded in another previous commit...

**Pull request merged**